splash/browser_tab.py

```diff
@@ -537,6 +537,13 @@ def http_get(self, url, callback, headers=None, follow_redirects=True):
             follow_redirects=follow_redirects
         )
 
+    def http_post(self, url, callback, headers=None, follow_redirects=True, body=None):
```

Comment:
ah, you're right, but this would require us to add some redirect middleware to Splash?